3-platform/shopping-cart/app/src/main/java/com/_6core/platform/shopping/cart/app/usecase/CheckoutUseCase.java

```diff
@@ -0,0 +1,11 @@
+package com._6core.platform.shopping.cart.app.usecase;
+
+import com._6core.lib.java.domain.model.cart.ShoppingCartV01;
+import java.util.List;
+
+public interface CheckoutUseCase {
+  // CheckoutShoppingCartResponse checkout(String cartId);
```

Comment:
usecase should have only one method, e.g. checkout

method of usecase will be invoked by rest/grpc adapters to force business flow execution